### DISCUSSION TITLE
ublock origin logger on firefox android(mobile devices)

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2782] [Ideas]
It is very inconvenient that close button shown in screenshot is at edge of touch screen...it is very close to refresh button marked ....can you move that buttons apart or place them not at edge of screen...problem is that sometimes other buttons are clicked & not the close

![Screenshot_20230822-090454-273.png](https://github.com/uBlockOrigin/uBlock-issues/assets/20338483/53613745-aaf7-44bb-aa3f-99fb2a6cc9b1)

**Comment by gorhill** [comment]
https://github.com/gorhill/uBlock/commit/eccfd3b823a128e5db24286bbb2de78251711ecf

**Comment by u-RraaLL** [comment]
I've got a similar issue.

The close button often overlaps/is adjoined to the view source button in the background, which can launch code viewer when you're trying to close the dialog.

Could you perhaps move the close button the the same row as the refresh button, which should make the  "drag" row act as a sort of padding above the button? @gorhill 

---

Alternatively, for mobile (or wherever the screen space is not enough) the source viewer button could be moved to inside the filter dialog > details > URL?

**Reply by u-RraaLL** [reply]
I'll consider this resolved since the docked dialog doesn't need to be closed.
